persistence-jpa/javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/persistence/jpa/JpaReadSideImpl.java

```diff
@@ -182,7 +182,7 @@ private void updateOffset(EntityManager entityManager, Offset offset) {
                 if (profile instanceof PostgresProfile) {
                     postgresqlBindUpdateOffsetQuery(updateOffsetQuery, sequenceOffset, timeUuidOffset);
                 } else {
-                    defaultBindUpdateOffsetQuery(updateOffsetQuery, sequenceOffset, timeUuidOffset);
+                    defaultBindUpdateOffsetQuery(profile, updateOffsetQuery, sequenceOffset, timeUuidOffset);
```

Comment:
Maybe, just to keep symmetry, we should have the following instead: 

```java
if (profile instanceof PostgresProfile) {
  postgresqlBindUpdateOffsetQuery(updateOffsetQuery, sequenceOffset, timeUuidOffset);

} else if (profile instanceof OracleProfile) {
  defaultBindUpdateOffsetQuery(
    updateOffsetQuery, 
    sequenceOffset == null ? "" : sequenceOffset, timeUuidOffset);
} else {
  defaultBindUpdateOffsetQuery(updateOffsetQuery, sequenceOffset, timeUuidOffset);
}
```

We have already an `if/else` branch testing the profile. I think it makes sense to extend it instead.
  
That will require to change the `defaultBindUpdateOffsetQuery` signature to: 
```java
private Query defaultBindUpdateOffsetQuery(Query query,  
                                           Object sequenceOffset, 
                                           String timeUuidOffset)
```

Reply:
👍 I agree with your suggestion.

Reply:
Thanks!

I should have said before, but it will be useful if we add a comment on the oracle branch explaining why we pass `""` when its `null`. That will help us understand it in a few months when we look back to this code.

Reply:
I added the comment with the link to issue and also added a null check for `timeUuidOffset`.

Reply:
I added 
* the comment with the link to issue
* a null check for `timeUuidOffset`
* link to StackOverflow in issue #1772